croud/organizations/auditlogs/commands.py

```diff
@@ -80,5 +80,5 @@ def auditlogs_list(args: Namespace) -> None:
         errors=errors,
         keys=["action", "actor", "created"],
         output_fmt=get_output_format(args),
-        transforms=[None, actor_id_transfor, None],
+        transforms={"actor": actor_id_transfor},
```

Comment:
Would you mind fixing the name of the transform method?

```suggestion
        transforms={"actor": actor_id_transform},
```

Reply:
Nice catch!